content/en/docs/setup/install/multicluster/shared-gateways/index.md

```diff
@@ -10,7 +10,7 @@ aliases:
 ---
 
 Follow this guide to configure a multicluster mesh using a
-[shared control plane topology](/docs/concepts/multicluster-deployments/#shared-control-plane-topology)
+[single control plane deployment](/docs/concepts/deployment-models/#single-control-plane)
```

Comment:
Sorry to be late to the discussion, but are we back with single control plane deployment instead of shared control plane? I thought we agreed once that in this topology we can have multiple control planes shared, for example, two clusters with a control plane and two clusters without a control plane. cc @costinm

Reply:
Oh yes, I missed this one. There's currently a mixture of the term "shared" and "single" in this PR, in several places. This example should definitely be called "shared control plane deployment", but see my other comment about the need to describe shared control plane in the concepts.

Reply:
@frankbu @vadimeisenbergibm  The content added by this PR does not mix `shared` and `single`. Quite the contrary, I made an additional effort to be consistent in the PR and use only `single` even when updating the links. It is beyond the scope of this PR to make all content outside of the concept consistent. That would need to be its own effort post 1.3.

Reply:
@rcaballeromx, I think you still have at least one link to `single` that is titled `shared`.

That said, those links (from the install section) should be titled `shared` and we need to either add a section in the concepts to describe `shared` or change the `single control plane` section to explain what `shared` means. 

We've had many discussions about this and concluded that `single control plane` is not a realistic Istio production configuration (even for single cluster), so we shouldn't describe it as a proper multicluster installation. That's why we updated the setup instructions to not say `single`. This PR should not undo that decision.

Reply:
@frankbu I'll wait until we reach consensus on the control plane section in the concept and implement the word choice we establish there. Does that SGTY?

Reply:
Makes sense to hold off, but I think it really needs to be changed before the doc goes live.